app/controllers/media_controller.rb

```diff
@@ -0,0 +1,54 @@
+class MediaController < ApplicationController
+  before_action only: [:show, :edit, :update, :destroy, :vote] { @medium = Medium.find(params[:id]) }
```

Comment:
Instead of using `before_action only:` here you can use `before_action except` since you're defining `@medium` for all but 3 of your controller methods.